compiler/src/org.graalvm.compiler.java/src/org/graalvm/compiler/java/BytecodeParser.java

```diff
@@ -1148,8 +1148,8 @@ protected void cleanupFinalGraph() {
     }
 
     /**
-     * Creates the frame state after the start node of a graph for an {@link IntrinsicContext
-     * intrinsic} that is the parse root (either for root compiling or for post-parse inlining).
+     * Creates the frame state after the start node of a graph for an {@link IntrinsicContext}
+     * intrinsic that is the parse root (either for root compiling or for post-parse inlining).
```

Comment:
Why this fix? The original is perfectly valid according to https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html

Reply:
Ok, my version of IntelliJ did not recognize that as a link. I thought was a mistake